crates/ra_ide_db/src/imports_locator.rs

```diff
@@ -20,7 +20,7 @@ impl<'a> ImportsLocator<'a> {
         Self { sema: Semantics::new(db) }
     }
 
-    pub fn find_imports(&mut self, name_to_import: &str) -> Vec<ModuleDef> {
+    pub fn find_imports(&mut self, name_to_import: &str) -> Vec<Definition> {
```

Comment:
I think this type is too loose, it includes things like locals, which can't be imported. I'd add an separate enum here with two variants here, or just an `Either`.

Reply:
Had had the first version with `Either`, but what always confuses me in it is that it spreads all over the codebase like a cancer, including the public API.

In this case, in order to be able to use the API from the `ra-assists`,  we either need to either reexport it here or add a dependency to the assists  module.

But I'll use it, whatever.